src/lib/operators/table_scan.hpp

```diff
@@ -53,22 +53,11 @@ class TableScan : public AbstractReadOnlyOperator {
     size_t chunk_scans_sorted{0};
 
     void output_to_stream(std::ostream& stream, DescriptionMode description_mode) const override {
-      if (chunk_scans_skipped == 0 && chunk_scans_sorted == 0) {
-        return;
-      }
+      OperatorPerformanceData<AbstractOperatorPerformanceData::NoSteps>::output_to_stream(stream, description_mode);
 
       const auto* const separator = description_mode == DescriptionMode::MultiLine ? "\n" : " ";
-      stream << separator << "Chunks: ";
-      if (chunk_scans_skipped > 0) {
-        stream << chunk_scans_skipped << " skipped";
-      }
-      if (chunk_scans_skipped > 0 && chunk_scans_sorted > 0) {
-        stream << ", ";
-      }
-      if (chunk_scans_sorted > 0) {
-        stream << chunk_scans_sorted << " scanned using binary search";
-      }
-      stream << ". ";
+      stream << separator << "Chunks: " << chunk_scans_skipped << " skipped, ";
```

Comment:
How do you feel about "early out" instead of skipped?

Reply:
Fine with me. Both for variable and printing? Can't think of a good name ... `chunk_scans_early_out`?

Reply:
`num_early_out_chunks`?

Only if you agree with it though. I have no strong opinion about it, just thought it might be slightly clearer.

Reply:
I am superbad at naming and when that makes it for somebody else clearer, I am happy with it.

And `num_sorted_scan_chunks`?

Reply:
Forgot about that one. Maybe `num_chunks_with_binary_search` and `num_chunks_with_early_out`?

Reply:
Done.